Source/UnitTests/VideoCommon/VertexLoaderTest.cpp

```diff
@@ -50,8 +48,12 @@ class VertexLoaderTest : public testing::Test
     memset(input_memory, 0, sizeof(input_memory));
     memset(output_memory, 0xFF, sizeof(input_memory));
 
-    memset(&m_vtx_desc, 0, sizeof(m_vtx_desc));
-    memset(&m_vtx_attr, 0, sizeof(m_vtx_attr));
+    m_vtx_desc.low.Hex = 0;
```

Comment:
This can be shortened to:

```suggestion
    m_vtx_desc = {};
```